### ISSUE TITLE
Reactor Computer Port missing documentation

### ISSUE THREAD

**Issue by fatboychummy** [BiggerSeries/BiggerReactors#23]
As the title states.

I help a lot in a discord server for computercraft and etc, and a common thing I've noticed is people rolling in stating that bigger reactors has changed the functions that are used, but that there is no docs for it.

Basic documentation would be nice to have, or if some already exists could you point me to it?

**Comment by gizmocodes** [comment]
Yeah, currently our documentation is lacking because none of us have taken the time to really work on the wiki. Currently, until documentation is written for it, the best way to see what functions there are is literally by looking at the source (which isn't ideal, but it works for now).

Inside of the the source files, each function that can be called will have an `@LuaFunction` above it. These are the three peripherals defined for CC usage: [Reactor](https://github.com/BiggerSeries/BiggerReactors/blob/1.16/src/main/java/net/roguelogix/biggerreactors/multiblocks/reactor/deps/ReactorPeripheral.java), [Turbine](https://github.com/BiggerSeries/BiggerReactors/blob/1.16/src/main/java/net/roguelogix/biggerreactors/multiblocks/turbine/deps/TurbinePeripheral.java), [Heat Exchanger](https://github.com/BiggerSeries/BiggerReactors/blob/1.16/src/main/java/net/roguelogix/biggerreactors/multiblocks/heatexchanger/deps/HeatExchangerPeripheral.java). As an example, (this is from memory) you should be able to do `connected()` as a function for the Reactor Computer Port, and it should return true or false depending on whether the reactor is properly assembled. Rogue can correct me if needed, since he was the one who wrote the peripherals (as well as most of the mod).

We hope to have the wiki properly updated with an easily navigable page at some point in the future, but it just hasn't happened yet. If you would like, me and Rogue can provide more support in the Discord for more usage on the various functions. I'mma leave this issue up though because it _is_ something we need to do for the wiki.

**Comment by Wolf1596Games** [comment]
Is there any way I can get a link to that Discord? I'm trying to modify a program that was written a few years ago for BigReactor control but there are some functions that don't exist/aren't working properly and I'd like some help.

**Comment by RogueLogix** [comment]
its on curseforge

**Comment by fatboychummy** [comment]
> Is there any way I can get a link to that Discord? I'm trying to modify a program that was written a few years ago for BigReactor control but there are some functions that don't exist/aren't working properly and I'd like some help.

We're the [Minecraft Computer Mods](https://discord.gg/H2UyJXe) discord server.

**Comment by Rising-Galaxy** [comment]
How to use them?
e.g.
local reactor = peripheral.find("BiggerReactors_Reactor")
local fuel = reactor.fuel()
if I wanna use the fuel() function, it is right?
but it tell me: attempt to call field 'fuel'

**Comment by Mitchell5200** [comment]
@Rising-Galaxy There's a pending PR to the BiR wiki, in it is some documentation about the computer port,
Post from BiR discord below

until the PR for the wiki update gets merged, https://github.com/BiggerSeries/Wiki/blob/9105d75e764791baaf858dd614663223c38cabfd/biggerreactors/CCintegration.md

OR, the direct code links
https://github.com/BiggerSeries/BiggerReactors/blob/1.18/src/main/java/net/roguelogix/biggerreactors/multiblocks/reactor/deps/ReactorPeripheral.java
https://github.com/BiggerSeries/BiggerReactors/blob/1.18/src/main/java/net/roguelogix/biggerreactors/multiblocks/turbine/deps/TurbinePeripheral.java
https://github.com/BiggerSeries/BiggerReactors/blob/1.18/src/main/java/net/roguelogix/biggerreactors/multiblocks/heatexchanger/deps/HeatExchangerPeripheral.java

**Comment by Rising-Galaxy** [comment]
Thank you. This has been very helpful to me
